### DISCUSSION TITLE
Streaming loading.tsx not working on work laptop and works fine on personal laptop

### DISCUSSION THREAD

**Discussion by arcria** [vercel/next.js#70763] [Help]
### Summary

I was working on the dashboard learn app from the vercel site. Chapter 9 the loading.tsx does not display the loading div. This is happening on a work laptop but I tried the same thing and it works fine on my personal laptop. Not sure what is going on. I created a simple test project using create-next-app and added a folder, with a loading.tsx and page.tsx, put in a setTimeout in my page for 10 seconds and it shows the loading component on my personal laptop. I copied zipped up the project and used the same one on my work laptop and same issue, just shows blank white until after 10 seconds. Both machines are windows 10, node, react and next.js are all the same versions. Has to be something with a firewall or something configured form work causing the streaming issue. Any ideas?

**Comment by AMHConnect** [comment]
Hi @arcria , did you by any chance find out what was causing this?
I'm having the exact same issue - and it's very frustrating!
Loading.tsx / Suspense works perfectly in CodeSandbox, and when I open my localhost website in BrowserStack, but just refuses to work normally in browser (Chrome/Firefox/Edge etc).

**Reply by arcria** [reply]
Hi @AMHConnect, in my case it was Sophos network software on my work computer that was actually blocking the Loading.tsx/suspense from working. Peripheral access control was blocking the stream on my windows laptop. Once IT removed my use from this group it worked fine, seems I was mistakenly placed in this group while co-workers were not so they were unaffected.

**Comment by caroparo** [comment]
Hi @AMHConnect,
Did you manage to resolve this issue? What are the relevant antivirus policies? I'm facing a similar issue where Kaspersky Endpoint Security on my work laptop prevents simple, localhost nextjs loading.js/suspense from working. Everything seems to be buffered and only shows up after suspense work's done. In Chrome DevTools network panel, the http request/response timing was all attributed to "waiting for server response" and not "content download". 
If it's something auto-blocked on client-side, doesn't it mean that production users might see this kind of blocked responses if they have similar antivirus installed?

**Comment by stj-be** [comment]
For me using an HTTPS connection worked by using `next dev --experimental-https`. It seems contents are not buffered on HTTPS connections.
